### PR TITLE
add no game found info when no game was found

### LINKED ISSUE #190: Show information when no game was found in Analysis
Show information that no game was found on the pages:
- `/analysis/latest` & `/analysis/:param`
- `/analysis`

### PR DESCRIPTION
**Pull request by maracuja-juice**
closes #190

### CODE REVIEW

**Comment by RohrerF** [comment]
image

on this page the text looks a little strange. 

Maybe an other text?
Something like no recently played games

**Comment by maracuja-juice** [comment]
Good point. I will change that.
Should I add the text as a new prop?

**Comment by maracuja-juice** [comment]
I did that now. Let me know what you think.

File: src/views/Analysis/NoGameFound.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Typography, TypographyT } from '@rmwc/typography';
+import styles from './NoGameFound.module.css';
+import classNames from 'classnames';
+
+interface NoGameFoundProps {
+  tag: string;
+  center?: boolean;
+  className?: string;
+  children?: any;
```
Comment (RohrerF) [inline src/views/Analysis/NoGameFound.tsx]: is string possible here instead of any?

Comment (maracuja-juice) [reply src/views/Analysis/NoGameFound.tsx]: Yes

**Pull request merged**